clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts

```diff
@@ -71,11 +78,14 @@ describe.skip("Config wizard with plus settings", () => {
         const { body } = interception.request;
         expect(body.length).to.eql(numSystems);
       });
-      /* This will redirect to localhost:3000/datamap instead of localhost:4000/datamap
-       * which is not actually what we want.
-       * However, it is difficult to test across zones, so all we can do is assert the path is right
-       */
-      cy.url().should("contain", "datamap");
+      cy.getByTestId("systems-classify-table");
```

Comment:
I tried running the tests by removing the `describe.skip` and enabling the feature flag to make the page appear. The test fails here, looking like this:
<img width="591" alt="Screen Shot 2022-11-11 at 15 17 11" src="https://user-images.githubusercontent.com/2236777/201443230-33f7c6c1-561e-469a-aab4-9e750a31ec40.png">

Reply:
I'm still getting an error just on `Allows calling the runtime scanner with classify`